src/blas_like/level3/Gemm/NN_Multistream.hpp

```diff
@@ -176,14 +181,13 @@ void SUMMA_NNB_impl_multistream(
         auto A1 = A1_STAR_MC.emplace(A1_STAR_MC.end(), g);
         auto D1 = D1Trans_MR_STAR.emplace(D1Trans_MR_STAR.end(), g);
 
-        auto const& stream_one = stream_pool.Next();
-        auto const& stream_two = stream_pool.Next();
+        auto const& the_stream = stream_pool.Next();
```

Comment:
We should synchronize the stream pool against `A` and `B`.

Reply:
No need, but I should sync it against `C`. I missed this one.

Reply:
Fixed.